src/clj/harja/palvelin/palvelut/yllapitokohteet/paallystys.clj

```diff
@@ -35,8 +35,7 @@
   (oikeudet/vaadi-lukuoikeus oikeudet/urakat-kohdeluettelo-paallystysilmoitukset user urakka-id)
   (let [vastaus (into []
                       (comp
-                        (map #(assoc % :tila (yllapitokohteet-domain/yllapitokohteen-tarkka-tila %)))
-                        (map #(assoc % :tila-kartalla (yllapitokohteet-domain/yllapitokohteen-tila-kartalla %))))
```

Comment:
Tässä oli nyt kaksi ongelmaa: päällystysilmoituksen tila ylikirjoitettiin kohteen tilaksi. Lisäksi näitä assoccauksia ei enää tarvi tehdä. Nämä tehtiin siksi että päällystysilmoitukset näytettiin kartalla ja nämä tiedot tarvittiin, nyt päällystysilmoitukset-välilehdellä näytetään suoraan kohteet kartalla.